pkg/k8s/from_yaml.go

```diff
@@ -14,11 +14,14 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// instead of pkg/maya/maya.go
+// Package k8s defines
+// all the k8s related
+// codebase
```

Comment:
Can we put these in a single line.

Reply:
Will do @sonasingh46 👍 thanks for the feedback.